tdp/core/variables/variables.py

```diff
@@ -62,56 +110,132 @@ class VariablesDict(MutableMapping):
         variables["key1"] = "value1" # set the value `value1` to the key `key1`
         value = variables["key1"] # get the value at the key `key1`
         del variables["key1"] # deletes value at key `key1`
+
+    Attributes:
+        _content: Content of the var file.
+        _name: Name of the var file.
     """
 
-    def __init__(self, content, name=None):
-        """
+    def __init__(self, content: Dict, name: Optional[str] = None):
+        """Initializes the VariablesDict instance.
+
         Args:
-            content ([Dict]): Content of a var file
+            content: Content of a var file.
+            name: Name of the var file. Defaults to None.
         """
         self._content = content
         self._name = name
 
     @property
-    def name(self):
+    def name(self) -> Optional[str]:
+        """Name of the var file.
+
+        Returns:
+            Name of the var file.
+        """
         return self._name
 
-    def copy(self):
+    def copy(self) -> Dict:
+        """Returns a copy of the content.
+
+        Returns:
+            Copy of the content.
+        """
         return self._content.copy()
 
-    def merge(self, mapping):
+    def merge(self, mapping: Dict) -> None:
+        """Merges the provided mapping into the content.
+
+        Args:
+            mapping: Mapping to merge.
+        """
         self._content = merge_hash(self._content, mapping)
 
-    def __getitem__(self, key):
+    def __getitem__(self, key: str) -> Any:
+        """Returns the item with the given key.
```

Comment:
Why document this kind of method (method like `__method__`) ? Theses methods are not called directly.

Reply:
I removed it too, I only left the comments on the `__init__` methods (as suggested in the Google Python Style Guide).